content/docs/details/entity-push-api/index.md

```diff
@@ -0,0 +1,205 @@
+
+## Fetch your AWS accounts
+
+Using the aws cli make sure you have it setup and configured. You can read more [here](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-welcome.html)
+
+```bash
+aws organizations list-accounts
+```
+
+<details>
+<summary>Errors</summary>
+
+If you encounter the following error make sure you have the proper permissions configured for yourself and check if you are using your correct AWS_PROFILE
+
+`An error occurred (AccessDeniedException) when calling the ListAccounts operation: You don't have permissions to access this resource.`
+
+</details>
+
+This will result in a response like
+
+```json
+{
+  "Accounts": [
+    {
+      "Arn": "arn:aws:organizations::111111111111:account/o-exampleorgid/111111111111",
+      "JoinedMethod": "INVITED",
+      "JoinedTimestamp": 1481830215.45,
+      "Id": "111111111111",
+      "Name": "Master Account",
+      "Email": "bill@example.com",
+      "Status": "ACTIVE"
+    },
+    {
+      "Arn": "arn:aws:organizations::111111111111:account/o-exampleorgid/222222222222",
+      "JoinedMethod": "INVITED",
+      "JoinedTimestamp": 1481835741.044,
+      "Id": "222222222222",
+      "Name": "Production Account",
+      "Email": "alice@example.com",
+      "Status": "ACTIVE"
+    },
+    {
+      "Arn": "arn:aws:organizations::111111111111:account/o-exampleorgid/333333333333",
+      "JoinedMethod": "INVITED",
+      "JoinedTimestamp": 1481835795.536,
+      "Id": "333333333333",
+      "Name": "Development Account",
+      "Email": "juan@example.com",
+      "Status": "ACTIVE"
+    },
+    {
+      "Arn": "arn:aws:organizations::111111111111:account/o-exampleorgid/444444444444",
+      "JoinedMethod": "INVITED",
+      "JoinedTimestamp": 1481835812.143,
+      "Id": "444444444444",
+      "Name": "Test Account",
+      "Email": "anika@example.com",
+      "Status": "ACTIVE"
+    }
+  ]
+}
+```
+
+Now you can use either this list directly and send the data into the Roadie catalog via your preferred way, using plain curl or any programming language you prefer. In the next section I'll show you a full example with nodejs
+
+### Listing the accounts and sending the entities to Roadie
+
+- We will use the official AWS javascript SDK. Make sure you have it available in your system
+- Make sure you have your `ROADIE_API_TOKEN` available in your shell
+- Make sure you are authenticated towards your AWS.
+- Make sure you are using your correct `AWS_PROFILE`
+
+We are going to fetch the available AWS accounts then pick the data we want to put on the Resource entities and finally send these into Roadie. We are going to use the `PUT /api/catalog/roadie-entities/sets/${setId}` endpoint. Provide a set id something descriptive. This provides the ability to issue subsequent requests towards the same set id and it will update all of the entities provided in the request body. This performs a full mutation, every entity in the set will be replaced by the new incoming entities in the new requests.
```

Comment:
```suggestion
We are going to fetch the available AWS accounts and then pick the data we want to put in the Resource entities and finally send these into Roadie. We are going to use the `PUT /api/catalog/roadie-entities/sets/${setId}` endpoint. To use this we have to provide a set id, which should be something descriptive. This provides the ability to issue subsequent requests towards the same set id and it will update all of the entities provided in the request body. This performs a full mutation, every entity in the set will be replaced by the new incoming entities in the new requests.
```